spec/models/validation_requestable_spec.rb

```diff
@@ -375,6 +375,7 @@
       end
 
       it "creates audit with correct information" do
+        travel_to(5.minutes.from_now)
```

Comment:
[I normally put travel_to in a block.](https://api.rubyonrails.org/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-travel_to). 

Will the time get reset at the end of the block automatically?

```
travel_to(5.minutes.from_now) do
  ... blah blah
end
```

Reply:
Yes, rubocop now objects to blocks because the time is reset automatically ☺️

Reply:
Can you demo that rubocop error ... I'm trying to get it and can't

        it "validates presence for consultation_summary" do
          travel_to(5.minutes.from_now) do
            blah = "dd"
          end
        end

Reply:
Oh dear!

So this is what I was thinking of: https://docs.rubocop.org/rubocop-rails/cops_rails.html#railsredundanttravelback

But in fact in this project we seem to have used `travel_back` at the end of several tests. So I must be misunderstanding something 🤔

Reply:
Well it isn't introducing a bug which is most important. I didn't know it now travel_back automatically ... so I'm educated. May be we are out of date with rubocop??

I'm updating rubocop-rspec as we speak ... see if people like the updates...